src/seedsigner/models/seed.py

```diff
@@ -239,3 +239,61 @@ def seedqr_supported(self) -> bool:
     @property
     def bip85_supported(self) -> bool:
         return False
+
+
+
+class ShamirSeed(Seed):
+    def __init__(self,
+                 mnemonics: List[str] = None,
+                 passphrase: str = "",
+                 wordlist_language_code: str = SettingsConstants.WORDLIST_LANGUAGE__ENGLISH) -> None:
+        self._wordlist_language_code = wordlist_language_code
+
+        if not mnemonics:
+            raise Exception("Must initialize a ShamirSeed with a mnemonic List[str]")
+        #TODO: save just one share. Extendable flag must be supported in embit.
+        self._mnemonics: List[str] = mnemonics
+
+        self._passphrase: str = ""
+        self.set_passphrase(passphrase, regenerate_seed=False)
+
+        self.seed_bytes: bytes = None
+        self._generate_seed()
+
+
+
+    @staticmethod
+    def get_wordlist(wordlist_language_code: str = SettingsConstants.WORDLIST_LANGUAGE__ENGLISH) -> List[str]:
+        if wordlist_language_code == SettingsConstants.WORDLIST_LANGUAGE__ENGLISH:
+            return slip39.SLIP39_WORDS
+        else:
+            raise Exception(f"Unrecognized wordlist_language_code {wordlist_language_code}")  
+        
+
+    def _generate_seed(self):
+        try:
+            share_set = slip39.ShareSet([slip39.Share.parse(m) for m in self._mnemonics])
+            self.seed_bytes = share_set.recover(self._passphrase.encode('utf-8'))
+        except Exception as e:
+            logger.info(repr(e), exc_info=True)
+            raise InvalidSeedException(repr(e))
+
+
+    @property
+    def seedqr_supported(self) -> bool:
+        return False
+
+
+    @property
+    def bip85_supported(self) -> bool:
```

Comment:
Can you add a TODO note on the `Seed` parent class that these should have been prefixed with "is_"?